src/test/resources/application.properties

```diff
@@ -1,2 +1,2 @@
-provider.nbp-api.base-url=http://localhost:8081
-app.base-currency=PLN
\ No newline at end of file
+provider.nbp-api.base-url=http://api.nbp.pl/api
```

Comment:
If the values are the same as the one in `src/main/resources/application.properties`, then there is no point of overriding.